### PR TITLE
Downgrade 'terraform-provider-openstack' to v1.15.0 in Runtime Provisioner

### PR DESCRIPTION
**Pull request by rafalpotempa**
**Description**

Changes proposed in this pull request:

- `terraform-provider-openstack` is downgraded to version v1.15.0.
- Version of `gophercloud` dependency is adjusted so it works with `terraform-provider-openstack`.

**Related issues**

Issue: https://github.com/kyma-incubator/compass/issues/1011

Kyma image bump: https://github.com/kyma-project/kyma/pull/7685

### CODE REVIEW

File: components/provisioner/go.mod [hunk 1]
```diff
@@ -46,3 +46,39 @@ replace github.com/census-instrumentation/opencensus-proto v0.1.0-0.201812141439
 replace k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible => k8s.io/client-go v0.15.8-beta.1
 
 replace github.com/Azure/go-autorest v11.5.0+incompatible => github.com/Azure/go-autorest v13.3.2+incompatible
+
+replace golang.org/x/crypto v0.0.0-20180816225734-aabede6cba87 => golang.org/x/crypto v0.0.0-20200317142112-1b76d66859c6
```
Comment (clebs) [inline components/provisioner/go.mod]: I think here you can replace all versions in one line by not putting any version on the left side:
`replace golang.org/x/crypto => golang.org/x/crypto v0.0.0-20200317142112-1b76d66859c6`

Comment (rafalpotempa) [reply components/provisioner/go.mod]: True, it was too late. I didn't think of that :)

**Pull request merged**